src/content/plugins/limit-chunk-count-plugin.md

```diff
@@ -44,7 +44,7 @@ module.exports = {
 
 ### `minChunkSize`
 
-To keep chunk size above the specified limit by merging chunks that are smaller than the limit, use [MinChunkSizePlugin)[/plugins/min-chunk-size-plugin]
+Keeping chunk size above the specified limit is no longer a feature of this plugin. Use [MinChunkSizePlugin)[/plugins/min-chunk-size-plugin] instead.
```

Comment:
Worth mentioning a version since when that happened as a reference or is that sufficient enough? CC @montogeek

Reply:
It is enough